cookbook/security/pre_authenticated.rst

```diff
@@ -11,6 +11,14 @@ box, Symfony supports most authentication mechanisms.
 These requests are called *pre authenticated* requests because the user is already
 authenticated when reaching your application.
 
+.. caution::
+
+    :doc:`User impersonation </cookbook/security/impersonating_user>` is not
+    compatible with pre Authenticated firewalls. The reason is that
```

Comment:
@pasdeloup it's "complicated" to say if you are right or wrong. First, the title casing is arbitrary and some words are uppercased and others aren't. Moreover, the hyphen after prefixes is also arbitrary in some cases, so I don't know if it's written "preauth..." or "pre-auth..."